clients/java/src/main/java/io/camunda/zeebe/client/impl/command/CreateProcessInstanceWithResultCommandImpl.java

```diff
@@ -74,6 +74,7 @@ public CreateProcessInstanceWithResultCommandImpl(
     this.httpClient = httpClient;
     httpRequestConfig = httpClient.newRequestConfig();
     useRest = preferRestOverGrpc;
+    requestTimeout(requestTimeout);
```

Comment:
We need to be careful with this here, since this now also sets this in the request objects (which we didn't do previously). This implies the user set a dedicated request timeout but we just want to ensure it's set in the underlying technical communication, not as a request parameter. I hope that is not too confusing, it's a bit late already 😅

Reply:
This is not needed here right ?

Reply:
Not the method call, no. But the http config needs to get the request timeout. We need to do that here "manually", the method call adjusts too much.

Reply:
Synced offline:
* The method call here is fine. the gRPC request send the `requestTimeout` as well now but it's not used in the broker anyway.